bin/run_sql_task.py

```diff
@@ -0,0 +1,50 @@
+"""
+Task runner for tasks written as SQL files in directories.
+
+This is a general mechanism for running tasks defined in SQL, however it's
+currently only used to perform the aggregations and mappings required for
+reporting.
+"""
+
+import os
+from argparse import ArgumentParser
+
+import importlib_resources
+import sqlalchemy
+from psycopg2.extensions import parse_dsn
+
+from report.sql_tasks.sql_script import SQLScript
+
+TASK_ROOT = importlib_resources.files("report.sql_tasks") / "tasks"
+
+parser = ArgumentParser(
+    description=f"A script for running SQL tasks defined in: {TASK_ROOT}"
+)
+parser.add_argument("-t", "--task", required=True, help="The SQL task name to run")
+
+
+def main():
+    args = parser.parse_args()
+
+    dsn = os.environ["DATABASE_URL"].strip()
```

Comment:
This changes to:

- read from env instead of pyramid config
- directly use SQLA transactions instead of pyramid.tm